sqlmesh/dbt/basemodel.py

```diff
@@ -196,6 +199,7 @@ def relation_info(self) -> AttributeDict[str, t.Any]:
                 "schema": self.table_schema,
                 "identifier": self.table_name,
                 "type": relation_type.value,
+                "quote_policy": AttributeDict(self.quoting.dict()),
```

Comment:
If we expect to work correctly with jinja after deserialization we should probably update the following in the JinjaMacroRegistry: https://github.com/TobikoData/sqlmesh/blob/main/sqlmesh/utils/jinja.py#L184-L188

in order to support recursive AttributeDicts

Reply:
Made recursive.